scene/gui/rich_text_label.h

```diff
@@ -663,6 +663,13 @@ class RichTextLabel : public Control {
 	void _scroll_changed(double);
 	int _find_first_line(int p_from, int p_to, int p_vofs) const;
 
+	const real_t auto_scroll_speed = 2;
```

Comment:
This probably should be configurable and set to `EDSCALE * 2` for the editor log, otherwise scroll is a bit slow if you are using large scale.